### PR TITLE
Added custom time manager

### PR DESCRIPTION
**Pull request by Wouterz90**

### CODE REVIEW

File: content/panorama/scripts/custom_game/time_control.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+function setTimerValue(event:ClockTimeEvent) {
+    var mainPanel = $.GetContextPanel().GetParent()!.GetParent()!;
+    var gameTime = mainPanel.FindChildTraverse("GameTime") as LabelPanel;
+
+    $.Msg(  event.seconds,event.minutes)
```
Comment (Perryvw) [inline content/panorama/scripts/custom_game/time_control.ts:5]: What is this formatting, also I guess this was for debugging and is no longer needed?

File: game/scripts/vscripts/TimeManager.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+        this.callbacks.forEach((x) => {
+            if (x.seconds == this.time) {
+                x.fn();
+            }
+        });
+    }
+
+    SetClockValue(seconds: number, minutes?: number) {
+        print(seconds, minutes);
```
Comment (Perryvw) [inline game/scripts/vscripts/TimeManager.ts:36]: dont think we need this print either

**Review by Perryvw — APPROVED** [review]

**Comment by Shushishtok** [comment]
Need to also resolve conflicts.

File: content/panorama/scripts/custom_game/time_control.ts (hunk 1 above)
Comment (Wouterz90) [reply content/panorama/scripts/custom_game/time_control.ts:5]: Not sure what you mean with the formatting

**Pull request merged**